pkg/systemd/generate/containers.go

```diff
@@ -321,9 +334,9 @@ func executeContainerTemplate(info *containerInfo, options entities.GenerateSyst
 		info.NotifyAccess = "all"
 		info.PIDFile = ""
 		info.ContainerIDFile = "%t/%n.ctr-id"
-		info.ExecStartPre = "/bin/rm -f {{{{.ContainerIDFile}}}}"
-		info.ExecStop = "{{{{.Executable}}}} stop --ignore --cidfile={{{{.ContainerIDFile}}}}"
-		info.ExecStopPost = "{{{{.Executable}}}} rm -f --ignore --cidfile={{{{.ContainerIDFile}}}}"
+		info.ExecStartPre = "/bin/rm \\\n\t-f {{{{.ContainerIDFile}}}}"
+		info.ExecStop = "{{{{.Executable}}}} stop \\\n\t--ignore \\\n\t--cidfile={{{{.ContainerIDFile}}}}"
+		info.ExecStopPost = "{{{{.Executable}}}} rm \\\n\t-f \\\n\t--ignore \\\n\t--cidfile={{{{.ContainerIDFile}}}}"
```

Comment:
This also does not make it look better IMO

Reply:
What would you like it to look like?

Reply:
just remove your change

Reply:
Ok I am only doing ExecStart now.